refman/theory.tex

```diff
@@ -24,9 +24,9 @@ \chapter{Library Development: Set Theory}
     \begin{tabular}{l|c|l}
       {}                         & Math symbol       & Lisa Kernel             \\ \hline
       Set Membership predicate   & $\in$             & \lstinline$in(s,t)$     \\
-      Subset predicate           & $\subset$         & \lstinline$subset(s,t)$ \\
+      Subset predicate           & $\subseteq$       & \lstinline$subset(s,t)$ \\
       Empty Set constant         & $\emptyset$       & \lstinline$emptyset()$  \\
-      Unordered Pair constant    & $(\cdot, \cdot )$ & \lstinline$pair(s,t)$   \\
```

Comment:
In ZFC the axioms actually assert existence of unordered pair (i.e. the set that contains only s and t). The symbole should be $\lbrace \cdot, \cdot \rbrace. Could you just change that please ? :)